### PR TITLE
Mobile friendly pages

### PR DESCRIPTION
**Pull request by jaydhulia**
https://www.loom.com/share/d69a509ef3ff41ccbc94561876c4d10e

- [x] Display warning if used in portrait mode
- [x] Allow mobile users on landscape mode
- [x] Skip tutorial for mobile users

### CODE REVIEW

File: app/public/templates/landing_page.html [hunk 1]
```diff
@@ -136,6 +180,33 @@
...
+          );
+          modal.show();
+        }
+      }
     </script>
   </body>
 </html>
+
+{#<br />
+For the best experience, it is recommended you visit this page from a desktop
```
Comment (drbalar) [inline app/public/templates/landing_page.html:210]: do we need this?

**Review by drbalar — DISMISSED** [review]
looks good, tested out locally on computer 🚀

File: app/public/templates/landing_page.html (hunk 1 above)
Comment (jaydhulia) [reply app/public/templates/landing_page.html]: Whoops, removed!

**Review by drbalar — APPROVED** [review]

**Review by ahmedhamodi — APPROVED** [review]
Looks good. Looking forward to trying this out locally on my phone when it's deployed!

**Pull request merged**